x-pack/plugins/observability/server/lib/rules/custom_threshold/lib/create_condition_script.ts

```diff
@@ -19,7 +19,8 @@ export const createConditionScript = (threshold: number[], comparator: Comparato
   }
   if (comparator === Comparator.OUTSIDE_RANGE && threshold.length === 2) {
     return {
-      source: `params.value < params.threshold0 && params.value > params.threshold1 ? 1 : 0`,
+      // OUTSIDE_RANGE/NOT BETWEEN is the opposite of BETWEEN. Use the BETWEEN condition and switch the 1 and 0
+      source: `params.value > params.threshold0 && params.value < params.threshold1 ? 0 : 1`,
```

Comment:
Shall we change one of the API integration tests to is not between?

Reply:
Good point, @maryam-saeidi. I was thinking about adding a new test case for it. I will give it a try in the PR